src/Compilers/CSharp/Portable/Lowering/DiagnosticsPass_Warnings.cs

```diff
@@ -388,7 +399,7 @@ private void CheckVacuousComparisons(BoundBinaryOperator tree, ConstantValue con
             }
         }
 
-        private void CheckOr(BoundBinaryOperator node)
```

Comment:
SGTM

Overall change LGTM as well.

JaredPar from a phone
http://blog.paranoidcoding.com/
________________________________
From: AlekseyTs <notifications@github.com>
Sent: Friday, May 1, 2015 10:48:31 AM
To: dotnet/roslyn
Cc: Jared Parsons
Subject: Re: [roslyn] Report CS0675 warning for compound assignment operator. (#2416)


In src/Compilers/CSharp/Portable/Lowering/DiagnosticsPass_Warnings.cs<https://github.com/dotnet/roslyn/pull/2416#discussion_r29518866>:

> @@ -388,7 +399,7 @@ private void CheckVacuousComparisons(BoundBinaryOperator tree, ConstantValue con
>              }
>          }
>
> -        private void CheckOr(BoundBinaryOperator node)


@jaredpar<https://github.com/jaredpar> How does "CheckForBitwiseOrSignExtend" sound?

-
Reply to this email directly or view it on GitHub<https://github.com/dotnet/roslyn/pull/2416/files#r29518866>.